gcp-cloud-run/README.md

```diff
@@ -2,7 +2,7 @@
 
 This module deploys Piiano Vault on a given GCP region. It will output the private Vault URL post deployment.
 
-This application version is 1.8.2 and is compatible with Vault version 1.8.2.
+This application version is 1.0.2 and is compatible with Vault version 1.8.2 .
```

Comment:
Is the last space needed?

Reply:
It is for the SED to work properly and it is also like that in the other cases. I am sure we can verify and modify if needed to work without the space, nonetheless, don't think it worth the effort